### PR TITLE
Add border when focused with keyboard

### PR DESCRIPTION
**Pull request by pajong**
### This PR will...
Add border when focused with keyboard

### Why is this Pull Request needed?
It is not apparent which element is focused. We want the viewer to know what element is focused when using keyboard.

### Are there any Pull Requests open in other repos which need to be merged with this?
https://github.com/jwplayer/jwplayer-plugin-related/pull/365

#### Addresses Issue(s):

JW8-2330

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/js/view/controls/components/button.js [hunk 1]
```diff
@@ -21,9 +22,7 @@ export default function (icon, apiAction, ariaText, svgIcons) {
 
     // Prevent button from being focused on mousedown so that the tooltips don't remain visible until
     // the user interacts with another element on the page
```
Comment (pajong) [inline src/js/view/controls/components/button.js]: How do we want to handle the tooltip?

**Comment by vseventer** [comment]
Tricky problem - one thing I noticed is that the ad skip button does not have an outline when navigating with keyboard.

**Comment by karimMourra** [comment]
@pajong I notice that the tooltips have the focus ring, is that expected behavior ? @aliciahurst do you have a preference? The AC says "Tooltips and other text decorations should also appear, but cannot occlude the visibility of the focus ring" which i do not interpret as requesting a focus ring around the tooltip. 
I also notice that the seek bar and menu items do not have focus rings, is that desired behavior ? @aliciahurst please take http://player-pr-test-jenkins.longtailvideo.com/builds/18697/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=player_ui/player_ui_with_the_kitchen_sink&suite=Player%20UI%20and%20Customization for a spin

**Comment by karimMourra** [comment]
@pajong also there is no focus rings around the related items in the overlay

File: src/js/view/controls/components/button.js [hunk 2]
```diff
@@ -19,11 +20,8 @@ export default function (icon, apiAction, ariaText, svgIcons) {
         });
     }
 
-    // Prevent button from being focused on mousedown so that the tooltips don't remain visible until
```
Comment (karimMourra) [inline src/js/view/controls/components/button.js]: I'm confused by the tooltip part of this comment. Are we sure the behavior is being respected ?

**Comment by pajong** [comment]
> Tricky problem - one thing I noticed is that the ad skip button does not have an outline when navigating with keyboard.

@vseventer Do you have the tag? I see border around skip on my end

**Comment by pajong** [comment]
> @pajong also there is no focus rings around the related items in the overlay

@karimMourra It already has a white border when it is focused. Do we want to have an extra browser border on top of that?

**Comment by aliciahurst** [comment]
Agree with Karim's points:
* Outline should not include tooltips
* Outline missing in menu items
* Outline should be on overlays even when there is an existing border (for consistency I prefer it)

**Comment by pajong** [comment]
@karimMourra Still working on removing the tooltip border, but the rest is in place. Let me know if you find anything else. Thanks!

**Pull request closed without merging**